modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -0,0 +1,204 @@
+{{#hasHttpSignatureMethods}}
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+from six.moves.urllib.parse import urlencode, urlparse
+from Crypto.PublicKey import RSA, ECC
+from Crypto.Signature import PKCS1_v1_5, pss, DSS
+from Crypto.Hash import SHA256, SHA512
+from base64 import b64encode
+from email.utils import formatdate
+
+class HttpSignatureHandler(object):
+    """OpenAPI HTTP signature handler to generate and verify HTTP signatures.
+
+    NOTE: This class is auto generated by OpenAPI Generator.
+    Ref: https://openapi-generator.tech
+    Do not edit the class manually.
+
+    :param configuration: .Configuration object for this client
+    """
+
+    ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
+    ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
+    ECDSA_KEY_SIGNING_ALGORITHMS = {ECDSA_MODE_FIPS_186_3, ECDSA_MODE_DETERMINISTIC_RFC6979}
```

Comment:
How about setting these parameters as module constants?
how about naming them with prefixes like SCHEME_ and ALGORITHM_
That way our users could use magic string constants like so, and not have to worry about typing quotes:
```
from petstore_api import signing

# Configure API client with HTTP signature authentication:
conf = petstore_api.Configuration(
  key_id='my-key-id',
  private_key_path='rsa.pem',
  signing_scheme=signing.SCHEME_HS2019,
  signing_algorithm=signing.ALGORITHM_PSS,
  signed_headers=['(request-target)', '(created)', 'host', 'date', 'Content-Type', 'Digest']
)
```

Reply:
How do you feel about making constants for the headers too, like:
```
HEADER_REQUEST_TARGET = '(request-target)'
HEADER_CREATED = '(created)'
HEADER_HOST = 'host'
HEADER_DATE = 'date'
HEADER_CONTENT_TYPE = 'Content-Type'
HEADER_DIGEST = 'Digest'
```

Reply:
done.